drafts/configure-logs-JSON-format-190010.adoc

```diff
@@ -127,80 +132,37 @@ For more information:
 
-* <<JMOT,Jaeger for Microprofile OpenTracing>>
-* <<hc21,MicroProfile Health Check 2.1>>
-* <<hm21,MicroProfile Metrics 2.1>>
+* <<hc21, Provide your own health check procedures>>
+* <<hm21,Monitor microservice applications easily wth metrics (mpMetrics-2.1)>>
 
 These early implementations are not available in 19.0.0.10 but you can try them out in our daily Docker image by running `docker pull openliberty/daily`. Let us know what you think!
-
-[#JMOT]
-== Jaeger for MicroProfile OpenTracing (jaeger-1.0)
-
-Open Liberty has the support of MicroProfile OpenTracing. A sample tracer was provided link:https://github.com/WASdev/sample.opentracing.zipkintracer[here] for using Zipkin as a tracing backend. With the introduction of `jaeger-1.0`, developers can also use Jaeger as a tracing backend.
-
-You will need to download the Jaeger client version 0.34.0 library and its dependencies from the link:/https://mvnrepository.com/artifact/io.jaegertracing/jaeger-client/0.34.0[Maven repository.]
-
-To enable the feature in the `server.xml` file:
-
-. Add `<feature>jaeger-1.0</feature>` under the `<featureManager>` section.
-
-. Add the following snippet:
-
-[source, xml]
-
-<library id="jaegerLib" apiTypeVisibility="api,ibm-api,spec,stable,third-party" >
-        <file name="<path>/jaegerLib_0.34/gson-2.8.2.jar" />
-        <file name="<path>/jaegerLib_0.34/jaeger-client-0.34.0.jar" />
-        <file name="<path>/jaegerLib_0.34/jaeger-core-0.34.0.jar" />
-        <file name="<path>/jaegerLib_0.34/jaeger-thrift-0.34.0.jar" />
-        <file name="<path>/jaegerLib_0.34/jaeger-tracerresolver-0.34.0.jar" />
-        <file name="<path>/jaegerLib_0.34/libthrift-0.12.0.jar" />
-        <file name="<path>/jaegerLib_0.34/slf4j-api-1.7.25.jar" />
-        <file name="<path>/jaegerLib_0.34/slf4j-jdk14-1.7.25.jar" />
-        <file name="<path>/jaegerLib_0.34/opentracing-util-0.31.0.jar" />
-        <file name="<path>/jaegerLib_0.34/opentracing-noop-0.31.0.jar" />
-    </library>
-
-. Define your application:
-
-[source, xml]
-
- <webApplication location="yourapp.war" contextRoot="/yourapp">
-        <!-- enable visibility to third party apis -->
-        <classloader commonLibraryRef="jaegerLib"
-            apiTypeVisibility="api,ibm-api,spec,stable,third-party" />
-    </webApplication>
-
-For the `JAEGER_PASSWORD`, the password can be encoded using the `securityUtility` command.
-
-Depending on Jaeger's sampling settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, Jaeger may not report every span created by the applications.
-
-For more information:
-
-* link:/https://github.com/jaegertracing/jaeger-client-java/blob/10c641f8df6316f1eac4d5b1715513275bcd724e/jaeger-core/README.md[Set up Jaeger settings using environment variables]
-
 [#hc21]
-== MicroProfile Health Check 2.1 (mpHealth-2.1)
+== Provide your own health check procedures (mpHealth-2.1)
```

Comment:
The bit in brackets, can you spell out the full feature name? "(MicroProfile Health Check 2.1)"